sklearn/datasets/tests/test_20news.py

```diff
@@ -88,3 +93,48 @@ def test_20news_normalization(fetch_20newsgroups_vectorized_fxt):
 
     assert_allclose_dense_sparse(X_norm, normalize(X))
     assert np.allclose(np.linalg.norm(X_norm.todense(), axis=1), 1)
+
+
+def test_20news_as_frame(fetch_20newsgroups_vectorized_fxt):
+    pd = pytest.importorskip('pandas')
+
+    bunch = fetch_20newsgroups_vectorized_fxt(as_frame=True)
+    check_as_frame(
+        bunch, partial(fetch_20newsgroups_vectorized_fxt),
```

Comment:
O_o why do we need `partial` if we're not setting any parameters to the function?